projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -113,3 +113,73 @@ export async function jobdDataRequest(): Promise<void> {
     );
   }
 }
+
+/**
+ * Sent a POST request to the server to get more information about the notebook that is failing the job run
+ * Returns the path to the notebook file and the index of the cell that is failing in the notebook
+ */
```

Comment:
I was looking into PR's description, to find out where the error message is being displayed. May you add a screenshot of the stracktrace/error visualised?

Reply:
@ivakoleva if you are talking about the job run error message, you can see the screen recording that I have added to the PR, it is visualised after the run and there I click "Get failing cell"

Reply:
I remember now I saw it in Fri, the video looks very good 😊

Reply:
@ivakoleva yes, a lo file is created with all the logs, so if the user wants to see more information they can just check the log file

Reply:
Is there a link in the log, where I can click and navigate to the failint cell in the notebook accordingly?

Reply:
no, the log file is just an txt, consisting of the standard logs we get on the CLI nothing additional is added there